src/libs/Permissions.ts

```diff
@@ -1,8 +1,9 @@
+import {OnyxEntry} from 'react-native-onyx';
 import CONST from '@src/CONST';
 import Beta from '@src/types/onyx/Beta';
 
-function canUseAllBetas(betas: Beta[]): boolean {
-    return betas?.includes(CONST.BETAS.ALL);
+function canUseAllBetas(betas: OnyxEntry<Beta[]>): boolean {
+    return !!betas?.includes(CONST.BETAS.ALL);
```

Comment:
This change is not required

Reply:
I think it is , without that return type will be `boolean | undefined` because `betas` can be undefined but I think we want always to return boolean in this case

Reply:
```js
function canUseAllBetas(betas: Beta[]): boolean {
    return betas.includes(CONST.BETAS.ALL);
}
```
This will be fine, and ts is also not complaining.